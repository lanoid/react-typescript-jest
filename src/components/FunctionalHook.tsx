import * as React from 'react';
import useWindowWith from '../hooks/WindowWidth';

const FunctionalHook: React.FunctionComponent = () => {
    const width = useWindowWith();
    
    return(
        <div>{ width }</div>
    );
}

export default FunctionalHook;