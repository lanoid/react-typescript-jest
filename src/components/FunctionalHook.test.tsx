import * as React from 'react';
import { shallow } from 'enzyme';
import FunctionalHook from './FunctionalHook';

describe('<FunctionalHook />', () => {
    const functionalHook = shallow(<FunctionalHook />);
    
    it('matches snapshot', () => {
        expect(functionalHook).toMatchSnapshot();
    });
});