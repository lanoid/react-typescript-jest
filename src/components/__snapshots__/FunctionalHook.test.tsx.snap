// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FunctionalHook /> matches snapshot 1`] = `
<div>
  1024
</div>
`;
